playground/theme-editor-app/package.json

```diff
@@ -6,6 +6,9 @@
     "type": "app"
   },
   "dependencies": {
-    "@jpmorganchase/theme-editor": ">=1.0.0"
+    "@jpmorganchase/theme-editor": "*",
+    "@jpmorganchase/uitk-core": "*",
+    "@jpmorganchase/uitk-lab": "*",
+    "@jpmorganchase/uitk-theme": "*"
```

Comment:
may also need icons?

Reply:
That's not being directly used in the app, probably in the `theme-editor` package